mcs/class/System/System_xtest.dll.sources

```diff
@@ -27,7 +27,7 @@ System/RemoteExecutorTests.cs
 ../../../external/corefx/src/Common/tests/System/Net/Configuration.cs
 ../../../external/corefx/src/Common/tests/System/Net/Configuration.Http.cs
 ../../../external/corefx/src/Common/tests/System/Net/Configuration.Certificates.cs
-#../../../external/corefx/src/System.Net.Requests/tests/*.cs
+#../../../external/corefx/src/System.Net.Requests/tests/*.cs:AuthenticationManagerTest.cs,FtpWebRequestTest.cs
```

Comment:
this line is still commented out, was it intentional?

Reply:
Yes, we're still getting random failures which might be due to problems with their Azure endpoint.